src/sections/location/location-table-head.tsx

```diff
@@ -0,0 +1,75 @@
+import Box from '@mui/material/Box';
+import TableRow from '@mui/material/TableRow';
+// import Checkbox from '@mui/material/Checkbox';
+import TableHead from '@mui/material/TableHead';
+import TableCell from '@mui/material/TableCell';
+import TableSortLabel from '@mui/material/TableSortLabel';
+
+import { visuallyHidden } from './utils';
+
+// ----------------------------------------------------------------------
+
+type UserTableHeadProps = {
+  orderBy: string;
+  order: 'asc' | 'desc';
+  onSort: (id: string) => void;
+  headLabel: Record<string, any>[];
+  // onSelectAllRows: (checked: boolean) => void;
+};
+
+export function UserTableHead({
+  order,
+  onSort,
+  orderBy,
+  headLabel,
+  // onSelectAllRows,
+}: UserTableHeadProps) {
+  return (
+    <TableHead>
+      <TableRow>
+        {/* <TableCell padding="checkbox">
+          <Checkbox
+            indeterminate={numSelected > 0 && numSelected < rowCount}
+            checked={rowCount > 0 && numSelected === rowCount}
+            onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+              onSelectAllRows(event.target.checked)
+            }
+          />
+        </TableCell> */}
```

Comment:
remove unnecessary comments
if this is useful for future reference then leave it